ideascube/serveradmin/catalog.py

```diff
@@ -35,6 +35,32 @@ def rm(path):
         shutil.rmtree(path)
 
 
+def load_from_file(path):
+    """Load catalog data from a local file
```

Comment:
> The fact it returns None is the important bit.

Agreed. This should definitely be in the docstring or comment.

> Also, if we don't want these to be exposed as API, should I make them private?

I'm talking about creating an API *documentation*, which we don't want nor need to do ;) That's fine to let the importable, even if we don't use them elsewhere.